test/xpu/xpu_op_db.yaml

```diff
@@ -0,0 +1,123 @@
+backend: XPU
```

Comment:
```suggestion
```

Reply:
can I just remove the backend filed in yaml file? It is not used actually.